### PR TITLE
Aci release 3.10

### PR DESCRIPTION
**Pull request by Vikash082**

### CODE REVIEW

File: roles/aci/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- name: include lib_openshift
+  include_role:
+    name: lib_openshift
+- name: Create namespaces
+  oc_project:
+    state: present
+    name: "{{ item }}"
+  with_items: "{{ annotate_namespace_list }}"
+  delegate_to: "{{ groups.oo_first_master.0 }}"
+- name: Get aci policy tenant name
```
Comment (vrutkovs) [inline roles/aci/tasks/main.yml]: Use `yedit`, similar to
```
  - name: Read master storage backend setting
    yedit:
      state: list
      src: /etc/origin/master/master-config.yaml
      key: kubernetesMasterConfig.apiServerArguments.storage-backend
    register: _storage_backend
```

Comment (Vikash082) [reply roles/aci/tasks/main.yml]: Hi @vrutkovs thank you for review. Does yedit works with yaml having multiple document ?

Comment (Vikash082) [reply roles/aci/tasks/main.yml]: Hi @vrutkovs , I tried yedit, but look to me it doesn't support reading from yaml file which has multiple document. We are trying to read the value of key from a deployment file which has multiple documents.

Comment (vrutkovs) [reply roles/aci/tasks/main.yml]: Ah, I see. Okay, lets use shell here then

**Comment by Vikash082** [comment]
Thank you @vrutkovs for review.

**Pull request merged**